qutebrowser/extensions/loader.py

```diff
@@ -118,6 +121,42 @@ def _walk_normal() -> typing.Iterator[ExtensionInfo]:
         yield ExtensionInfo(name=name)
 
 
+def walk_extensions() -> typing.Iterator[ExtensionInfo]:
+    """Walk external extensions."""
+    ext_dir = os.path.join(standarddir.data(), 'extensions')
+    if not os.path.exists(ext_dir):
+        try:
+            os.mkdir(ext_dir)
+        except OSError:
+            pass
+        return
+
+    if any(os.listdir(ext_dir)):
+        log.extensions.warning(
+            "Loading EXTENSIONS in {}: this in an "
+            "EXPERIMENTAL FEATURE.".format(ext_dir)
+        )
+    else:
+        return
+
+    prefix = 'qutebrowser.extensions.third_party.'
+    for finder, name, _ispkg in pkgutil.walk_packages(
+            path=[ext_dir],
+            prefix=prefix,
+            onerror=_on_walk_error):
+        if name not in sys.modules:
+            try:
+                # Import the module with the finder so that it is in
+                # sys.modules readif for _load_component.
+                finder.find_module(name).load_module(name)
+            except Exception:
+                log.extensions.exception(
+                    "Failed importing extension: {}".format(name[len(prefix):])
```

Comment:
```suggestion
                    "Exception while importing extension: {}".format(name[len(prefix):])
```

(or "Failed to import")